src/OmniSharp.Abstractions/Models/v2/CodeFoldingBlock.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace OmniSharp.Models.V2
+{
+    public class CodeFoldingBlock
+    {
+        public CodeFoldingBlock(Range textSpan, string type)
+        {
+            Range = textSpan;
+            Type = type;
+        }
+
+        /// <summary>
+        /// The span of text to collapse.
+        /// </summary>
+        public Range Range { get; }
+
+        /// <summary>
+        /// If the block is one of the types specified in <see cref="CodeFoldingBlockKind"/>, that type.
+        /// Otherwise, null.
+        /// </summary>
+        public string Type { get; }
+    }
+
+    public class CodeFoldingBlockKind
+    {
+        public static readonly string Comment = nameof(Comment).ToLowerInvariant();
```

Comment:
I chose to include these values because the VS Code folding API has special handling for these types of blocks